src/Command/BuildDocsCommand.php

```diff
@@ -34,17 +34,17 @@ class BuildDocsCommand extends Command
 {
     protected static $defaultName = 'build:docs';
 
-    private $buildContext;
+    private $buildConfig;
     private $missingFilesChecker;
     /** @var SymfonyStyle */
     private $io;
 
-    public function __construct(BuildContext $buildContext)
+    public function __construct(BuildConfig $buildConfig)
```

Comment:
What are you planning for this command? should we still keep it in order to debug or as well to be a preview tool for doc contributors?

In that case, shouldn't we leverage the new `DocBuilder` in this command?

Reply:
Good question. I didn't want to remove the command or the PHAR-related things in this PR to minimize changes. But yes, we can remove this in the future because Symfony won't use it.

Reply:
don't you plan any preview tool for doc contributors?

Reply:
You are right. We need to do that ... but to be completely honest, that's not a priority for us right now. It's enough if this project is usable in the GitHub CI to alert us when an error happens. We can work on the "doc preview" later ... and I'd like to make it as similar as possible to the doc published on symfony.com. I think we'll be able to make that 👍